pretix_extended_api/views/tickets.py

```diff
@@ -46,3 +51,43 @@ def attendee_has_ticket(self, request, **kwargs):
 
         qs = qs.filter(events_filter)
         return Response({"user_has_admission_ticket": qs.exists()})
+
+    @scopes_disabled()
+    @action(url_path="attendee-tickets", detail=False, methods=["get"])
+    def attendee_tickets(self, request, *args, **kwargs):
+        check_permission(request, "can_view_orders")
+
+        serializer = AttendeeTicketBodySerializer(data=request.query_params)
+        serializer.is_valid(True)
+
+        attendee_email = serializer.data["attendee_email"]
+
+        qs = OrderPosition.objects.filter(
+            attendee_email=attendee_email,
+            order__status=Order.STATUS_PAID,
+            item__admission=True,
+        )
+
+        items = []
+        for order_position in qs:
+            item = {
+                "price": str(order_position.price),
+                "item": {
+                    "name": order_position.item.name.data,
+                    "id": order_position.item.id,
+                },
+                "answers": [
+                    {
+                        "question_id": a.question.id,
+                        "answer": a.answer,
+                        "options": [
+                            {"identifier": o.identifier, "answer": o.answer.data}
+                            for o in a.question.options.all()
+                        ],
+                        "question": a.question.question.data,
+                    }
+                    for a in order_position.answers.all()
```

Comment:
```suggestion
                    for answer in order_position.answers.all()
```

I think we should use more meaningful names than `a` and `o`